src/bob/pipeline/internals.clj

```diff
@@ -333,10 +334,10 @@
 
 (m/defstate pipeline-status-change-connection
   :start (let [datasource    (doto (PGDataSource.)
-                               (.setServerName states/db-host)
-                               (.setPort states/db-port)
-                               (.setDatabaseName states/db-name)
-                               (.setUser states/db-user))
+                               (.setServerName (get-in config [:postgres :host]))
```

Comment:
if these names are defined with defonce, can be referred from the config ns

Reply:
I changed it to defonce postgres maybe you like it more that way ;)